lib/osut/utils.rb

```diff
@@ -252,31 +251,41 @@ def genConstruction(model = nil, specs = {})
 
     case specs[:type]
     when :shading
-      mt = :sheathing
+      mt = :material
       d  = 0.015
       a[:compo][:mat] = @@mats[mt]
       a[:compo][:d  ] = d
       a[:compo][:id ] = "OSut|#{mt}|#{format('%03d', d*1000)[-3..-1]}"
     when :partition
-      d  = 0.015
-      mt = :drywall
-      a[:clad][:mat] = @@mats[mt]
-      a[:clad][:d  ] = d
-      a[:clad][:id ] = "OSut|#{mt}|#{format('%03d', d*1000)[-3..-1]}"
+      unless specs[:clad] == :none
+        d  = 0.015
+        mt = :drywall
+        a[:clad][:mat] = @@mats[mt]
+        a[:clad][:d  ] = d
+        a[:clad][:id ] = "OSut|#{mt}|#{format('%03d', d*1000)[-3..-1]}"
+      end
 
-      # TO DO: replace sheathing by mineral below a certain Uo factor. 
-      mt = :sheathing
+      d  = 0.015
+      d  = 0.100      if specs[:frame] == :medium
+      d  = 0.200      if specs[:frame] == :heavy
+      d  = 0.100      if u
```

Comment:
This allows one to switch from un/insulated partitions, massive 1x layer construction, etc.